buildroot-external/board/nabucasa/green/patches/linux/0001-arm64-dts-rockchip-Add-NabuCasa-Green-board.patch

```diff
@@ -231,16 +238,16 @@ index 000000000000..e38fbfd50156
 +&i2c0 {
 +	status = "okay";
 +
-+	vdd_cpu: rk8601@41 {
++	vdd_cpu: fan53555@41 {
 +		compatible = "rockchip,rk8601";
 +		reg = <0x41>;
 +		vin-supply = <&vcc5v0_sys>;
-+		regulator-compatible = "rk860x-reg";
```

Comment:
This property is deprecated, and can be removed. The driver will match on the compatible string.